AFQ/api.py

```diff
@@ -182,14 +188,48 @@ def __init__(self,
         b0_threshold : int, optional
             The value of b under which it is considered to be b0. Default: 0.
 
+        target_b_val : float, optional
+            Which b value you want to use from the dataset.
+            If None, all are used. Default: None
+
+        target_b_sensitivity : float, optional
+            Sensitivity of target_b_val.
+            Only used if target_b_val is not None.
+            B values selected will be in the range
+            target_b_val +/- target_b_sensitivity
+            Default: 50
+
         odf_model : string, optional
-            Which model to use for determining directions in tractography
-            {"DTI", "DKI", "CSD"}. Default: "DTI"
+            Which model to use for determining directions in tractography.
+            Can be one of: {"DTI", "DKI", "CSD"}. Default: "DTI"
 
         directions : string, optional
             How to select directions for tracking (deterministic or
             probablistic) {"det", "prob"}. Default: "det".
 
+        reg_subject : str or Nifti1Image, optional
+            The source image data to be registered.
+            Can either be a Nifti1Image, a path to a Nifti1Image, or
+            If "mni_T2", "b0", "dti_fa_subject", "dti_fa_template",
+            "subject_sls", "hcp_atlas", "mni_T1", or "power_map,"
+            image data will be loaded automatically.
+
+        reg_template : str or Nifti1Image, optional
+            The target image data for registration.
+            Can either be a Nifti1Image, a path to a Nifti1Image, or
+            If "mni_T2", "b0", "dti_fa_subject", "dti_fa_template",
+            "subject_sls", "hcp_atlas", "mni_T1", or "power_map,"
+            image data will be loaded automatically.
+            (defaults to the MNI T2)
+
+        mask_templ : bool, optional
+            Whether to mask the chosen template(s) with a brain-mask
+            Default: True
+
+        reg_algo : string, optional
+            Algorithm to use for registration.
+            Can be one of: {"syn", "slr"}. Default: "syn"
```

Comment:
If you are using `sls` and `hcp_atlas` this has to be SLR, right? And vice versa, if these are images it has to be "syn". So maybe we don't need user input?

Reply:
I now assume reg_algo and logger.error if the registration choices mix slr/syn. Should I throw an error instead of using logger.error?